nemo/collections/nlp/models/machine_translation/mt_enc_dec_model.py

```diff
@@ -481,10 +481,10 @@ def setup_training_data(self, train_data_config: Optional[DictConfig]):
         self._train_dl = self._setup_dataloader_from_config(cfg=train_data_config)
 
     def setup_multiple_validation_data(self, val_data_config: Union[DictConfig, Dict]):
-        self.setup_validation_data(self._cfg.get('validation_ds'))
+        self.setup_validation_data(val_data_config)
```

Comment:
Do we need setup_multiple_validation_data at all? It seems redundant.
@titu1994 what do you think?

Reply:
@michalivne Method `MTEncDecModel.setup_multiple_validation_data` overrides `ModelPT.setup_multiple_validation_data`, so if we remove  `MTEncDecModel.setup_multiple_validation_data` we will need to rewrite all dataloader setup.